scripts/sandbox.js

```diff
@@ -1,40 +1,66 @@
-let sandboxLevel = "";
-let sandboxObject = ObstacleCube;
-let sanboxTpX = 3;
-let sandboxTpY = 3;
-let sandboxPortalColor = "Blue";
+const sandbox = {
+  history: [null],
+  level: "",
```

Comment:
Is the level ever a string? How about saving the default value as a function that does nothing? `() => null`

Reply:
I refuse